src/main/resources/locales/cy/style-guide/iac-daily-list-additional-cases.json

```diff
@@ -0,0 +1,4 @@
+{
+    "title": "Rhestr Dyddiol y Siambr Mewnfudo a Lloches  – achosion ychwanegol",
```

Comment:
The casing for additional cases in welsh is different to the one below and to the English one

Reply:
Updated